partner_capital/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2015 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
```

Comment:
We have three copyright lines for the same company. Now this might be correct, but mostly copyright for code developed by employee's of a company goes to the company, so it might be more appropiate to mention the company here (and the authors under contributors). But that is for the people of tecnativa to answer.

Reply:
@NL66278 we always attribute to the person the corresponding copyright, so we prefer to put each one individually. It's true that the new "style" is: `Copyright <year> Tecnativa - <Person name>` for avoiding "spam" in mails, but it's not so important.

Reply:
@pedrobaeza Thanks for the explanation!